spec.html

```diff
@@ -19379,7 +19288,7 @@ <h1>
         <emu-alg>
           1. Let _env_ be GetThisEnvironment().
           1. Assert: _env_.HasSuperBinding() is *true*.
```

Comment:
Please add an assertion in `HasSuperBinding()` that if it returns true, the environment record is a function environment record.